spec/integration/hanami/helpers/escape_helper_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+RSpec.describe "Escape helper" do
+  subject do
+    Users::Show.new.call(format: :html, user: user).to_s
+  end
```

Comment:
Seeing this as the subject of a helper test – and having to look elsewhere to see how the helper is actually being used within the template – makes me think that it might be nice to support an "inline template" style of building views. I'll add that as to our Trello to consider for the future :)